src/Algebra/Bundles/Raw.agda

```diff
@@ -285,3 +285,33 @@ record RawLoop  c ℓ : Set (suc (c ⊔ ℓ)) where
 
   open RawQuasigroup rawQuasigroup public
     using (_≉_ ; ∙-rawMagma; \\-rawMagma; //-rawMagma)
+
+record RawKleeneAlgebra c ℓ : Set (suc (c ⊔ ℓ)) where
+  infix  8 _⋆
+  infixl 7 _*_
+  infixl 6 _+_
+  infix  4 _≈_
+  field
+    Carrier               : Set c
```

Comment:
Can we please get rid of the needles whitespace here?

Reply:
Done.